test/functional/txn_clone.py

```diff
@@ -8,12 +8,6 @@
 from test_framework.util import *
 
 class TxnMallTest(BitcoinTestFramework):
-
-    def __init__(self):
-        super().__init__()
-        self.num_nodes = 4
-        self.setup_clean_chain = False
```

Comment:
Sounds good. I've added a new commit that makes setting `self.num_nodes` mandatory. I haven't done the same for `self.setup_clean_chain`, since I think it makes more sense for that to have a default. If people feel strongly, it can be added in a future commit.